src/components/button/Button.stories.tsx

```diff
@@ -60,5 +63,21 @@ storiesOf("Button", module)
         <div>
             <Button key="link" link onClick={action("delete click")} icon={{shape: "trash"}}/>
             <Button key="link" link onClick={action("copy click")} icon={{shape: "copy"}}/>
+            <Button key="link" link onClick={action("window-close click")} icon={{shape: "window-close"}}/>
+            <Button key="link" link onClick={action("refresh click")} className="icon-button-border" icon={{shape: "refresh"}}/>
+            <Button key="link" link onClick={action("settings click")} className="icon-button-background" icon={{shape: "settings"}}/>
+        </div>
```

Comment:
any reason for adding "icon-button-border" and "icon-button-background" classes ? we already support icon button with border in clarity-react.

Reply:
updated the code.